### DISCUSSION TITLE
How to deal with the "extension.ini" files when installing extensions?

### DISCUSSION THREAD

**Discussion by RenaudILTIS** [datenstrom/community#920] [Ask a question] [answered]
Hello,
I apologize in advance for this beginner's question, but I've searched long without finding a clear answer.
It's probably obvious to professional coders.

I'm currently testing Yellow's themes :
1. I unzipped the Wittstock theme folder and copied all the files into the system/themes folder, next to the 5 Stockholm files.
2. I modified the theme parameter in system/extensions/yellow-system.ini
3. It works.

Now I'd like to install the Paris theme as well, without uninstalling the Wittstock theme.

My problem: I don't know how to proceed with the "extension.ini" file 🤔
Should I add the contents of the "extension.ini" file of the Paris theme after the contents of the "extension.ini" file already present?

Is there another way of doing this?
(Is it the same for extensions in the system/extensions folder?)

Thank you very much for this incredible work 👍

**Comment by GiovanniSalmeri** [comment, marked as answer]
It's easier than you imagine 😊 In order to install an extension (feature or language or theme) you have to download the ZIP file (**without** uncompressing it) and copy it into your `system/extensions` folder. When you refresh any page of the site in your browser, all the necessary files will be automagically unzipped in their proper place. `extension.ini` will be not copied, because it contains only the instructions for correctly installing the other files.

(And yes, Yellow is a wonderful work 👍)

**Reply by RenaudILTIS** [reply]
Okaaaayyyyy! I tried this method, but in the wrong directory : `system/themes` !
Now with `system/extensions`, it's working.
Thanks 👍

**Comment by schulle4u** [comment]
Just for clarification: All downloadable Yellow resources are classified as extensions, regardless whether they are features, themes or languages. The recommended installation for every extension consists of uploading the zip file into your `system/extensions` folder and opening your site in the browser. There are slightly nerdy installation methods for developers, but this one is what every well-documented extension should suggest. 🙂

**Reply by RenaudILTIS** [reply]
Thank you very much for this clarification. Indeed the word "extension" confused me. I can move forward now.
